phys2bids/interfaces/txt.py

```diff
@@ -6,12 +6,13 @@
 """
 
 import logging
+from collections import Counter
+from operator import itemgetter
 
 import numpy as np
 
 from phys2bids.physio_obj import BlueprintInput
-from operator import itemgetter
-from collections import Counter
+
```

Comment:
```suggestion
```

You don't need two blank lines here. You do before function statements, but not before the definition of e.g. "constants."